syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.hq.request";
option java_outer_classname = "UsOptionChainExpireDateRequestProto";

message UsOptionChainExpireDateRequest {
  string securityCode = 1;  //正股代码
}